gpii/node_modules/lifecycleManager/src/LifecycleManager.js

```diff
@@ -532,7 +542,7 @@ var gpii = fluid.registerNamespace("gpii");
     // (etc) that are relevant to the previous preferences. This result is not enough for applying new preferences.
     gpii.lifecycleManager.applyWithNewPrefs = function (that, session, contextualPrefs, prefsAppliedLocalEvent) {
         var newPrefsSet = {};
-        fluid.set(newPrefsSet, ["contexts", session.model.activeContextName, "preferences"], contextualPrefs);
+        fluid.set(newPrefsSet, ["contexts", session.model.activePrefsSetName, "preferences"], contextualPrefs);
```

Comment:
It's odd to see expressions like `["contexts", session.model.activePrefsSetName, "preferences"]` where one name has been renamed to use "prefsSet" as `activePrefsSetName` while its parent is still named as `contexts`. This situation repeatedly shows up in the code base.

To sync up all names, the ideal way is to rename all "context" to "prefsSet". But doing so will need a database upgrade to make this change for all preference safe documents in CouchDB.

Or we stick with the old name of "context".

What do you think, @amb26?